en/04_Changelogs/5.0.0.md

```diff
@@ -523,6 +525,13 @@ This is a major release and contains many breaking API changes. Deprecation warn
   - In Silverstripe CMS 4, calling [`SQLSelect::setLimit()`](api:SilverStripe\ORM\Queries\SQLSelect::setLimit()) with `0` as argument would unset the limit. In Silverstripe CMS 5, it sets a limit of `0` causing the query to return no results. Call `setLimit()` with `null` to unset the limit.
 - [`ArrayList::limit()`](api:SilverStripe\ORM\ArrayList::limit()) in Silverstripe CMS 5 throws an `InvalidArgumentException` when called with a negative `$length` or `$offset`.
 
+### Dynamic properites {#dynamic-properties}
```

Comment:
```suggestion
### Dynamic properties
```
No need to add an explicit anchor if it's just the same as the default

Reply:
Updated